src/test/java/com/github/iveina2021/api_test/GetAllProductsListTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.iveina2021.api_test;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class GetAllProductsListTest {
+
+    @BeforeEach
+    public void setUp() {
+        RestAssured.baseURI = "https://automationexercise.com";
+    }
+
+    @Test
+    public void getAllProductsList() {
+        Response response = RestAssured.given()
+                .get("/api/productsList");
+        String responseText = response.getBody().prettyPrint();
+
+        response.then()
+                .statusCode(200);
+
+        Assertions.assertThat(responseText).contains("products");
```

Comment:
This check is very weak, the response text may contain products in any place and this test will become green by mistake.

Options to improve:
1) Use jsonPath to check response structure i.e. /products or something like that
2) Convert products to list of objects (and check them properly)

Reply:
Fixed it together